gateway/1.4/modules/ROOT/pages/_partials/task-node-affinity.adoc

```diff
@@ -0,0 +1,243 @@
+
+[[set_node_affinity]]
+== Set Node Affinity for a Pod
+
+The Kubernetes scheduler can use node affinity rules to match nodes to Pods. The matching process enables the scheduler to select the node into which a Pod is deployed.
+
+Configure node affinity rules within a Pod spec. A Pod spec is YAML file in which you can specify rules for required, preferred, or both types of matching:
+
+* `requiredDuringSchedulingIgnoredDuringExecution`: The node must match all node affinity rules configured for a Pod when the Kubernetes scheduler applies this type of matching. Otherwise, the deployment fails. 
+* `preferredDuringSchedulingIgnoredDuringExecution`: Based on weighted preferences set for a Pod, the Kubernetes scheduler selects the best matching node. The scheduler must select a node, so if no node matches any of the preferences, the scheduler selects a node that does not match. 
+
+For guidance, see the following configuration options:
+
+* <<strict_match_only>>
+* <<soft_match_only>>
+* <<all_match_types>>
+
+[[strict_match_only]]
+=== Configure a Required Node Matching Rule for a Pod 
+
+In required node matching, the node must meet all node affinity rules configured for a Pod. Otherwise, the deployment fails. 
+
+//end::temp-tag1[]
+
+//TAG for common node affinity settings
+//tag::config-flex-affinity-condition[]
+Configure a node affinity rule in the Pod spec file. Each condition to a rule requires a node label as a `key`, an operator, and one or more node label values to accept for a node with that key. For operators, you can use `In`, `NotIn`, `Exists`,  `DoesNotExist`, `Gt`, and `Lt`.
+//end::config-flex-affinity-condition[]
+
+[source,yaml,subs=attributes+]
+----
+affinity:
+ nodeAffinity:
+   requiredDuringSchedulingIgnoredDuringExecution:
+     nodeSelectorTerms:
+     - matchExpressions:
+       - key: <node-label-key>
+         operator: <operator>
+         values:
+         - <node-label-value>
+       # - Add any other node label values. 
+    # - Add any other required conditions.
+----
+
+You can add as many values and conditions as the node requires.
+
+After configuring the Pod spec, proceed to <<upgrade_helm_chart>>.
+
+//TAG for upgrading pod YAML spec
+//tag::upgrade-flex-pod-spec-yaml[]
+[[upgrade_helm_chart]]
+== Upgrade the Helm Chart
+
+After adding the node affinity settings to your Pod spec, incorporate the spec file into the Helm chart for your Flex Gateway Ingress Controller so that the Kubernetes scheduler can use the setting. https://helm.sh/docs/intro/install[Helm^] is a tool used to install Flex Gateway, monitoring tools, and applications. A minimum Helm version of 3.0.0 is required. 
+
+Use a Helm command to incorporate the node affinity settings from your Pod spec into the Helm chart. The command to use depends on whether Flex Gateway is installed:
+
+* If you are installing Flex Gateway for the first time, use the command to set all values in the chart:
++
+[source,kubernetes,subs=attributes+]
```

Comment:
@ahejeij Does the multi-line syntax look ok below?

Reply:
yes!